scripts/translate-redirects.js

```diff
@@ -0,0 +1,82 @@
+// Copyright 2016-2020, Pulumi Corporation.  All rights reserved.
```

Comment:
There's not really any new code here -- I've just copied this coe from the Pulumi program in order to be able to run it as an individual script. (As mentioned in the PR description, I'll remove all dead code in a follow-up PR.)